File: src/main/resources/export/cpp/Subsystem-default_command.cpp
#foreach ($component in $components)
#if ("#type($component)" == "frc2::SubsystemBase" || "#type($component)" == "frc2::PIDSubsystem")
#foreach ($command in $commands)
#if($command.name == $component.getProperty("Default Command").value)
#set($len = $params.size() - 2)
#set($last = $len + 1)

#if ($command != "None")
        m_#required_subsystem($command)->SetDefaultCommand(#class($command.name)(m_#required_subsystem($command)));
#end
#end
#end
#end
#end

File: src/main/resources/export/java/Subsystem-default_command.java
#set($subsystem = $helper.getByName($subsystem_name, $robot))
#set($command = $subsystem.getProperty("Default Command").getValue())
#set($params = $subsystem.getProperty("Default command parameters").getValue())

#if ($command != "None" && $command != "")
        setDefaultCommand(#new_command_instantiation_nt( $command $params ));
#end
